angular/src/components/icon.component.ts

```diff
@@ -60,24 +52,24 @@ export class IconComponent implements OnChanges {
   }
 
   get iconCode(): string {
-    return IconMap[this.icon];
+    return BitwardenIcons.CharCodeMap[this.icon];
```

Comment:
We don't seem to actually be using iconCode anywhere, which would remove the need for BitwardenIcons constant.

Reply:
Removed the getter and subsequent util class